modules/ecs/ipfs/security_groups.tf

```diff
@@ -25,8 +25,15 @@ resource "aws_security_group" "alb_external" {
   }
 
   ingress {
-    from_port   = local.swarm_ws_port
-    to_port     = local.swarm_ws_port
+    from_port   = local.swarm_tcp_port
+    to_port     = local.swarm_tcp_port
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = var.use_ssl ? local.swarm_wss_port : local.swarm_ws_port
```

Comment:
If we're going to do this (which I like), can we just simplify the configuration everywhere?

e.g. [here](https://github.com/ceramicnetwork/terraform-aws-ceramic/pull/12/files#diff-9cbc8f16d40d68570a20be5b20b3377586f07fb87c914b3f9e8a5a7a4ba5a622R9)
```
  swarm_tcp_port   = 4010
  swarm_ws_port    = 4011
  swarm_wss_port   = 4012
```
becomes
```
  swarm_tcp_port   = 4010
  swarm_ws_port    = var.use_ssl ? 4012 : 4011
```

This way, the container def only needs one WS* port, there are fewer target groups, listeners, etc.